setup.py

```diff
@@ -93,7 +96,15 @@ def read(*rnames):
       extras_require={
           'zcml': ZCML_REQUIRES,
           'component': COMPONENT_REQUIRES,
-          'test': TESTS_REQUIRE,
```

Comment:
Side note: I find it super annoying if I have to scroll back up to the top to look up those definitions instead of spelling out the dependencies directly down here.

Reply:
I inlined all the requirements in https://github.com/zopefoundation/zope.location/pull/14/commits/03fa620cffc20d8ae1398141a776d9811fb30c55.